tests/test_param_routing.py

```diff
@@ -111,7 +118,7 @@ def build_fn_no_args(meta, compile_kwargs):
 
     clf = wrapper_class(model=build_fn_no_args)
     clf.fit(X, y)
-    assert set(clf.get_meta().keys()) == wrapper_class._meta - {"_user_params"}
+    assert wrapper_class._meta.issubset(set(clf.get_metadata().keys()))
```

Comment:
:-1:

Reply:
Idem above. Swapping out for hardcoded values.